tutorials/embedding/word2vec.py

```diff
@@ -42,7 +42,7 @@
 import numpy as np
 import tensorflow as tf
 
-import gen_word2vec as word2vec
+word2vec = tf.load_op_library(os.path.dirname(os.path.realpath(__file__)) + '/word2vec_ops.so')
```

Comment:
Do you need the absolute path here? You're already assuming you're in the same directory, why no load just 'word2vec_ops.so'?